src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/WorkflowTests/ChangeNotification.cs

```diff
@@ -89,10 +85,9 @@ public async Task Negatively_confirmed_and_then_changed_shows_notification()
             await ChangeApprenticeship(new ChangeBuilder(apprenticeship));
 
             var retrieved = await GetApprenticeship(apprenticeship);
-            retrieved.Should().BeEquivalentTo(new
-            {
-                DisplayChangeNotification = true,
-            });
+            retrieved.ChangeOfCircumstanceNotifications.HasFlag(ChangeOfCircumstanceNotifications.EmployerDetailsChanged).Should().BeTrue();
+            retrieved.ChangeOfCircumstanceNotifications.HasFlag(ChangeOfCircumstanceNotifications.ProviderDetailsChanged).Should().BeTrue();
+            retrieved.ChangeOfCircumstanceNotifications.HasFlag(ChangeOfCircumstanceNotifications.ApprenticeshipDetailsChanged).Should().BeTrue();
```

Comment:
FluentAssertions [has a handy](https://fluentassertions.com/enums/#enums) `Should().HaveFlag(..)` check

```suggestion
            retrieved.ChangeOfCircumstanceNotifications.Should().HaveFlag(ChangeOfCircumstanceNotifications.EmployerDetailsChanged);
            retrieved.ChangeOfCircumstanceNotifications.Should().HaveFlag(ChangeOfCircumstanceNotifications.ProviderDetailsChanged);
            retrieved.ChangeOfCircumstanceNotifications.Should().HaveFlag(ChangeOfCircumstanceNotifications.ApprenticeshipDetailsChanged);
```

Reply:
Didn't know that